server/pom.xml

```diff
@@ -12,7 +12,7 @@
 
   <properties>
     <version.org.jboss.modules>1.1.3.GA</version.org.jboss.modules>
-    <version.io.netty_netty>4.0.0.Alpha5</version.io.netty_netty>
+    <version.io.netty_netty>4.0.0.Alpha8</version.io.netty_netty>
```

Comment:
`4.x` which was used to build daemon on is still in Alpha. I'm not sure what implications are when we switch to 3.x which is last "stable" release.